doc/jetty-config.md

```diff
@@ -49,6 +49,13 @@ number of seconds, see http://man7.org/linux/man-pages/man7/socket.7.html.  Note
 that the effect of setting this option may vary depending upon the operating
 system's underlying implementation.
 
+### `idle-timeout-milliseconds`
+
+This optional setting can be used to control how long Jetty will allow a
+connection to be held open by a client without any activity on the socket.  If
+a connection is idle for longer than this value, Jetty will forcefully close
+it from the server side.  Jetty's default value for this setting is 30 seconds.
```

Comment:
Scratch my last comment on the idle timeout as it pertains to worker threads being idle.  That is actually controlled by the idle timeout passed into the `QueuedThreadPool` constructor:

https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/trapperkeeper-webserver-jetty9-1.1.1/src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj#L280

We're hard-coding that to 60 seconds right now, following what Jetty does if you call one of the other forms of the constructor that don't allow you to specify the idle timeout:

https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/trapperkeeper-webserver-jetty9-1.1.1/src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj#L145-L149

So, there doesn't seem to be anything relevant here to discuss for this setting.